README.md

```diff
@@ -71,7 +71,7 @@ The site will automatically refresh since it is bundled with livereload.
 ## Deploy a branch to surge.sh for testing and preview
 We use surge.sh to deploy directly from a branch to test new features and fixes. To do this:
 * Build `yarn build-staging`. You'll need to login to a surge account if this is the first you are running. You can create an account from the CLI.
-* Deploy `yarn deploy-staging`
+* Deploy `yarn deploy-surge`
```

Comment:
Hey, nice catch - really sorry about that @GregoryHorvath

Reply:
> Hey, nice catch - really sorry about that @GregoryHorvath

Haha no worries, I should've catched that before I ran it, it's all on me.